packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -2762,6 +2762,64 @@ describe('plugin-meetings', () => {
           assert.isOk(errorThrown);
         });
 
+        it('should send ICE_CANDIDATE_ERROR metric if media connection fails and ice candidate errors have been gathered', async () => {
+          let errorThrown = undefined;
+
+          meeting.roap.doTurnDiscovery = sinon.stub().returns({
+            turnServerInfo: undefined,
+            turnDiscoverySkippedReason: undefined,
+          });
+          meeting.meetingState = 'ACTIVE';
+          meeting.mediaProperties.waitForMediaConnectionConnected.rejects({iceConnected: false});
+
+          const forceRtcMetricsSend = sinon.stub().resolves();
+          const closeMediaConnectionStub = sinon.stub();
+          Media.createMediaConnection = sinon.stub().returns({
+            close: closeMediaConnectionStub,
+            forceRtcMetricsSend,
+            getConnectionState: sinon.stub().returns(ConnectionState.Connected),
+            initiateOffer: sinon.stub().resolves({}),
+            on: sinon.stub(),
+          });
+
+          meeting.iceCandidateErrors.set('701_error', 1);
```

Comment:
can we send 2 of the same errors and 1 other error so that we can see the metric being correctly counted per unique error?

Reply:
I am here not doing it through callback, but directly updating map content. I have added kind of error to check out if metrics are correctly populated. But if you think that we should populate map here through the callback, i can do it.